index.js

```diff
@@ -234,7 +256,7 @@ class Router {
 						});
 				}
 				if (actions.useNext) {
-					if (manifestedPaths.some((path) => path.test(request.pathname))) {
+					if (Array.from(manifest.values()).some((path) => path.test(request.pathname))) {
```

Comment:
Converting to an array for execution is a bit slower isn't it? (And this happens 1000x more often than the setup, which is of negligible performance  concern)